src/service/action-impl.js

```diff
@@ -492,7 +496,8 @@ export class ActionService {
     let n = target;
     while (n) {
       const actionInfos = this.matchActionInfos_(n, actionEventType);
-      if (actionInfos) {
+      const isEnabled = !matches(n, DISABLED_PSEUDOCLASS_);
```

Comment:
There could be, sure, but I would be very surprised since I tried a few of them. This is a native API that accesses the browser-computed value.

Compare with jQuery code: https://github.com/jquery/jquery/blob/2d4f53416e5f74fa98e0c1d66b6f3c285a12f0ce/external/sizzle/dist/sizzle.js#L463-L517 and https://github.com/jquery/jquery/blob/2d4f53416e5f74fa98e0c1d66b6f3c285a12f0ce/external/sizzle/dist/sizzle.js#L1709-L1771